ibm_cloud_sdk_core.gemspec

```diff
@@ -33,8 +33,8 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_runtime_dependency "concurrent-ruby", "~> 1.0"
-  spec.add_runtime_dependency "http", "~> 5.1.1"
-  spec.add_runtime_dependency "jwt", "~> 2.2.1"
+  spec.add_runtime_dependency "http", "~> 5.1"
+  spec.add_runtime_dependency "jwt", "~> 2.2"
```

Comment:
Agreed, this is pretty strict versioning.  If we need it in the future, we can do it then.